buildSrc/src/main/java/com/yelp/gradle/varanus/GlobalDependencies.kt

```diff
@@ -80,6 +80,6 @@ object BuildScriptLibs {
 
 object Publishing {
     const val GROUP = "com.yelp.android"
-    const val VERSION = "2.0.2"
+    const val VERSION = "2.0.4"
```

Comment:
Is there a reason we're skipping 3?

Reply:
For some reason it's at 2.0.3 internally, I think I bumped the version to try and fix something

Reply:
with regards to publishing internally, not in the library itself